core/view/src/parser/index.ts

```diff
@@ -251,35 +225,71 @@ export class Parser {
             }
 
             if (multiNode) {
-              children.push({
-                path: [...path, localKey],
-                value: multiNode,
-              });
+              return {
+                ...rest,
+                children: [
+                  ...children,
+                  {
+                    path: [...path, localKey],
+                    value: multiNode,
+                  },
+                ],
+              };
             }
           }
         } else if (localValue && typeof localValue === 'object') {
-          if (hasApplicability(localValue)) {
-            const applicabilityNode = this.parseApplicability(
+          const determineNodeType =
+            this.hooks.determineNodeType.call(localValue);
```

Comment:
I've noticed that `determineNodeType` alternates between being called w/ a _value_ (`localValue` here) and also w/ just the key name ([here](https://github.com/player-ui/player/pull/80/files#diff-96a718b00b84d926bdfa59f06fbc95d051ee90f066adb5b3bc742ffdf3299092R148))